apps/shooting-clubs/models/submission.js

```diff
@@ -1,12 +1,19 @@
 'use strict';
+const _ = require('lodash');
 
 module.exports = data => {
   const response = {};
+  const activity = [
+    { activity: 'new', response: 'Application' },
+    { activity: 'renew', response: 'Renewal' },
+    { activity: 'vary', response: 'Vary' }
+  ];
 
   response.AuthorityType = 'Shooting clubs';
-  response.ApplicationType = data.activity === 'new' ? 'Application' : 'Renewal';
+  response.ApplicationType = data.activity ?
+    _.find(activity, { activity: data.activity }).response : 'Renewal';
 
-  if (data.activity === 'renew') {
+  if (data.activity === 'renew' || data.activity === 'vary') {
     response['Cusomter.CustomerReference'] = data['reference-number'];
```

Comment:
`Cusomster?` looks like a typo

Reply:
ah thanks, thought I had got all of them. I've removed anyway, because these are no longer being used in the form.